src/assets/scripts/client/InputController.js

```diff
@@ -1002,6 +1003,38 @@ export default class InputController {
         );
     }
 
+    /**
+     * Retrieve and return the x/y canvas position where the user clicked in the provided event, where the
+     * x/y origin is the center of the canvas. All units are pixels.
+     *
+     * @for InputController
+     * @method _getMousePositionRelativeToCenterOfViewFromEvent
+     * @param event {jQuery Event}
+     * @return {object} in shape of `{x: -154, y: 388}`, in pixels
+     */
+    _getMousePositionRelativeToCenterOfViewFromEvent(event) {
+        const positionRelativeToCenterOfView = CanvasStageModel.translateMousePositionToCanvasPosition(
+            event.pageX, event.pageY
+        );
+
+        return positionRelativeToCenterOfView;
+    }
+
+    /**
+     * Retrieve and return the x/y relative position where the user clicked in the provided event
+     *
+     * @for InputController
+     * @method _getMouseRelativePositionFromEvent
+     * @param event {jQuery Event}
+     * @return {array<number>}
+     */
+    _getMouseRelativePositionFromEvent(event) {
+        const { x, y } = this._getMousePositionRelativeToCenterOfViewFromEvent(event); // ignores panning/zooming
+        const relativePosition = this._translatePointToKilometers(x, y);
```

Comment:
this `const` could be removed in favor of (adding an empty line 😉 ) and directly returning:
```js
return this._translatePointToKilometers(x, y);
```

Reply:
I'll do you one better... the whole method wasn't needed 😉